src/services/cloudfront/schema.graphql

```diff
@@ -18,7 +18,7 @@ type awsCloudfront implements awsBaseService @key(fields: "id") {
   origins: [awsCloudfrontOriginData]
   logging: awsCloudfrontLoggingConfig
   elb: [awsElb] @hasInverse(field: cloudfrontDistribution)
-  s3: [awsS3] @hasInverse(field: cloudfrontDistribution)
+  s3: [awsS3] @hasInverse(field: cloudfrontDistributions)
```

Comment:
to update these fields to plural, you need to update the corresponding `field` param in the connection objs. I may be missing something but I dont see those changes.

Reply:
Oh yes,  I've updated the field to plural in the s3 schema. Cloudfront is the one who has the connection to S3. So, in this case, the names in the schemas should be updated. The same for cloudtrail.